cookiecutter/tap-template/{{cookiecutter.tap_id}}/.github/workflows/{% if cookiecutter.include_cicd_sample_template == 'GitHub' %}ci_workflow.yml{%endif%}

```diff
@@ -15,15 +15,15 @@ jobs:
         python-version: [3.9]
 
     steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v3
     - name: Set up Python {{ '${{ matrix.python-version }}' }}
-      uses: actions/setup-python@v2
+      uses: actions/setup-python@v4
       with:
         python-version: {{ '${{ matrix.python-version }}' }}
     - name: Install Poetry
       run: |
         python -m pip install --upgrade pip
-        pip install poetry==1.2.*
+        pip install poetry==1.3.*
```

Comment:
We should probably use a constraints file do this can be kept up to date with dependabot. But that's for a future PR.